File: 240204/007/007.java
import java.io.*;

class Meeting {
    String code;
    char point;
    int time;
    public Meeting(String code, char point, int time) {
        this.code = code;
        this.point = point;
        this.time = time;
    }

    public void print() {
        System.out.println("secret code : " + this.code);
        System.out.println("meeting point : " + this.point);
        System.out.println("time : " + this.time);
    }
}

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] inputs = br.readLine().split(" ");
        Meeting meeting = new Meeting(inputs[0], inputs[1].charAt(0), Integer.parseInt(inputs[2]));
        meeting.print();
    }
}

File: 240316/007/007.py
code, place, time = input().split()

class Spy:
    def __init__(self, code, place, time):
        self.code = code
        self.place = place
        self.time = time

spy = Spy(code, place, time)
print('secret code :', spy.code)
print('meeting point :', spy.place)
print('time :', spy.time)